src/@types/Shortcut.ts

```diff
@@ -48,9 +48,6 @@ interface Shortcut {
   /** A readable, internal unique id. */
   id: ShortcutId
 
-  /** A function that returns true if the shortcut should be highlighted in the Toolbar. */
-  isActive?: (getState: () => State) => boolean
-
```

Comment:
`isActive` is used by other (non-formatting) shortcuts, so we cannot remove it.

e.g. Try toggling sort:

https://github.com/cybersemics/em/blob/a33035e2076d82d8114459d662b0508ca353d686/src/shortcuts/toggleSort.ts#L38-L45

Reply:
I added this back to the interface.